kafka.broker/k8s/templates/service-kafka.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             - /bin/sh
             - -c
             - /opt/bitnami/scripts/kafka/setup.sh && kafka-storage.sh format --config "${KAFKA_CONF_FILE}" --cluster-id "lkorDA4qT6W1K_dk0LHvtg" --ignore-formatted && /opt/bitnami/scripts/kafka/run.sh
-          image: "bitnami/kafka:3.1.0"
+          image: "bitnami/kafka:3.5"
```

Comment:
Please see previous feedback on version changes, handle them separately as we need to review any known issues with the releases before upgrading.

Reply:
I have to test all of the examples anyway so it saves some time to validate this works as well.